### ISSUE TITLE
CodeReview2 for HairStyle

### ISSUE THREAD

**Issue by zzx0921** [ZiranLi/HairStyle_Sprint2#28]
Your project looks pretty interesting and I tried to run your code on my Mac. The README file said that you will combine the two parts of your works together in Sprint3. I thought it would be in the folder named grabcutios-master. So I opened that project first.

First I tried to build the project and run it on my simulator. Unfortunately it didn't work very well. When I tried to register, I clicked the Register button but nothing happened. The same problem occurs when I tried to sign in. I don't know exactly the functions of each of your code so I can't tell what was wrong. I noticed that in your LoginInViewController.swift, a value "u" was defined but never used. Maybe there are something wrong with your signInButtonTapped function. Hope you can fix it later.

Then I tried to look at some details of your Github. However, the files of your Github are not well-organized. So it seems hard for people who don't know anything about your project to understand the architecture of your project. My suggestion is to make it more organized and maybe you can add more readme file to give more information about your code and your project. And try to tell people how to use or test your app and make it more understandable.

For your codes, they look amazing and I think they can be better if you add some comments in the beginning of each code to tell the main purpose of it and delete some useless lines of code.

In summary, your project looks amazing. I hope you can fix the problem and I'm looking forward to your final app.
